test/distrib/cpp/run_distrib_test_cmake.sh

```diff
@@ -24,49 +24,44 @@ apt-get update
 apt-get install -t jessie-backports -y libssl-dev
 
 # Install c-ares
-cd third_party/cares/cares
-git fetch origin
-git checkout cares-1_15_0
-mkdir -p cmake/build
-cd cmake/build
+mkdir -p "third_party/cares/cares/cmake/build" && pushd "$_"
```

Comment:
I think the `pushd "$_"` is slightly cryptic and I had to look it up (= other will likely have to as well). Readers of this script will be users trying to see what's the right way of installing gRPC with cmake, so having them look up meaning of commands is not ideal.
I suggest just doing `mkdir -p "third_party/cares/cares/cmake/build"` and `pushd "third_party/cares/cares/cmake/build"` because that's much more obvious.

Reply:
Also below.

Reply:
Done!